src/ixdat/tools.py

```diff
@@ -273,3 +277,20 @@ def tstamp_to_yyMdd(tstamp: float) -> str:
         year % 100, chr(ord("A") + month - 1), day
     )
     return date_string
+
+
+def tstamp_to_string(tstamp: float) -> str:
+    """Return a string representation if unix timestamps `tstamp`"""
+    dt = datetime.datetime.fromtimestamp(tstamp)
+    string_format = ixdat.config.config.timestamp_string_format
+    if string_format == "native":
+        # ixdat shows months as capital letters, where Jan->A, Feb->B etc.
+        month_letter = ascii_uppercase[dt.month - 1]
+        # Brings to the total format to: 22E18 14:34:55
+        string_format = f"%y{month_letter}%d %H:%M:%S"
+    return dt.astimezone(ixdat.config.config.timezone).strftime(string_format)
```

Comment:
This fails on Windows if `tstamp <= 86399`, i.e. for any time Jan 1 1970 or before. The problem is described here: https://stackoverflow.com/a/41400321
I only discovered it because some idiot (me) made tstamp default to 0 for the msrh_sec reader (where they don't care about the absolute time). I will make an Issue that that should be a better default. But might be worth finding a more robust time manipulation code here as well.

Reply:
Funny problem, as it turns out, the problem isn't `fromtimestamp` on 0.0, which works, but applying the timezone means that internally some c function is called again on a timestamp which is now negative, which isn't supported. As it turns out, creating the datetime with timezone to begin with works. So, this should be fixed the commit just pushed.